### PR TITLE
Add `@types/postcss-less`

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

This is a part of #4496 and improves type-checking about LESS. (fixing `// @ts-ignore`)

> Is there anything in the PR that needs further explanation?

See <https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/postcss-less>

### CODE REVIEW

File: lib/utils/isStandardSyntaxAtRule.js [hunk 1]
```diff
@@ -13,15 +13,13 @@ module.exports = function (atRule) {
...
-	if (atRule.mixin) {
+	if ('mixin' in atRule && atRule.mixin) {
 		return false;
 	}
 
 	// Ignore Less detached ruleset `@detached-ruleset: { background: red; }; .top { @detached-ruleset(); }`
 	if (
-		// @ts-ignore TODO TYPES Is this property really exists?
-		atRule.variable ||
+		('variable' in atRule && atRule.variable) ||
```
Comment (m-allanson) [inline lib/utils/isStandardSyntaxAtRule.js:22]: Is it ok to do this instead?

```suggestion
		'variable' in atRule ||
```

Or are there times where having both checks is needed?

Comment (ybiquitous) [reply lib/utils/isStandardSyntaxAtRule.js:22]: > Is it ok to do this instead?

Currently, yes.

But I'm worried that `'variable' in atRule` would not work if the implementation of `postcss-less` would change to `atRule.variable = false or true` and the type definition would not be changed.
So, I think it safer to check `atRule.variable`.

> Or are there times where having both checks is needed?

Rather, I'd like to remove checking `'variable' in atRule` if possible, but the TypeScript checker doesn't allow it. See the following error:

```
lib/utils/isStandardSyntaxAtRule.js:22:11 - error TS2339: Property 'variable' does not exist on type 'AtRule | ImportAtRule | VariableAtRule | MixinAtRule | FunctionAtRule'.
  Property 'variable' does not exist on type 'AtRule'.

22  	(atRule.variable) ||
    	        ~~~~~~~~
```

Does anyone know how to make a successful type-check without changing the runtime code?

Comment (ybiquitous) [reply lib/utils/isStandardSyntaxAtRule.js:22]: In addition, the following code seems easier to read (but out of the scope of this PR):

```js
// Ignore Less detached ruleset `@detached-ruleset: { background: red; }; .top { @detached-ruleset(); }`
if ('variable' in atRule && atRule.variable) {
	return false;
}
if (!atRule.nodes && atRule.raws.afterName === '' && atRule.params[0] === '(') {
	return false;
}
```

Comment (m-allanson) [reply lib/utils/isStandardSyntaxAtRule.js:22]: Thanks for the explanation!

> Does anyone know how to make a successful type-check without changing the runtime code?

I do not know this :(

**Review by m-allanson — APPROVED** [review]
:+1:

**Review by jeddy3 — APPROVED** [review]
Thank you!

**Pull request merged**